src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -547,54 +547,51 @@
         </Setter>
     </Style>
 
-    <Style x:Key="InputStyle" TargetType="TextBox">
+    <Style x:Key="InputStyle" TargetType="{x:Type TextBox}">
         <Setter Property="Margin" Value="0,0,0,12" />
         <Setter Property="MinWidth" Value="62px" />
-        <Setter Property="Cursor" Value="IBeam" />
```

Comment:
Is this removed intentionally?

Reply:
yup, since I have changed the internal structure of the style keeping the same functionality